examples/config.hocon.sample

```diff
@@ -89,47 +124,35 @@ elasticsearch {
   # - max-timeout: the maximum attempt time before a client restart
   # - ssl: if using the http client, whether to use ssl or not
   client {
-    endpoint: {{elasticsearchEndpoint}}
-    port: {{elasticsearchTransportPort}}
-    max-timeout: {{elasticsearchMaxTimeout}}
-    ssl: false
+    endpoint: "{{elasticsearchEndpoint}}"
+    port: "{{elasticsearchTransportPort}}"
+    maxTimeout: "{{elasticsearchMaxTimeout}}"
+    ssl: {{sslBool}}
   }
 
   # When using the AWS ES service
   # - signing: if using the http client and the AWS ES service you can sign your requests
   #    http://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html
   # - region where the AWS ES service is located
   aws {
-    signing: false
-    region: {{elasticsearchRegion}}
+    signing: {{signingBool}}
+    region: "{{elasticsearchRegion}}"
   }
 
   # index: the Elasticsearch index name
   # type: the Elasticsearch index type
   cluster {
-    name: {{elasticsearchClusterName}}
-    index: {{elasticsearchIndex}}
-    type: {{elasticsearchType}}
+    name: "{{elasticsearchClusterName}}"
+    index: "{{elasticsearchIndex}}"
+    clusterType: "{{elasticsearchType}}"
   }
 }
 
-# Events are accumulated in a buffer before being sent to Elasticsearch.
-# The buffer is emptied whenever:
-# - the combined size of the stored records exceeds byte-limit or
-# - the number of stored records exceeds record-limit or
-# - the time in milliseconds since it was last emptied exceeds time-limit
-buffer {
-  byte-limit: {{elasticsearchBufferByteThreshold}}
-  record-limit: {{elasticsearchBufferRecordThreshold}}
-  time-limit: {{elasticsearchBufferTimeThreshold}}
-}
 
-# Optional section for tracking endpoints
+# Section for tracking endpoints
 monitoring {
-  snowplow {
-    collector-uri: "{{collectorUri}}"
-    collector-port: 80
-    app-id: {{sinkKinesisAppName}}
-    method: GET
-  }
```

Comment:
could you leave the snowplow bit? This makes it clearer that we're using a snowplow collector to do the monitoring